frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/checklist_cell/checklist_cell_editor.dart

```diff
@@ -276,6 +249,24 @@ class _ChecklistItemState extends State<ChecklistItem> {
       ),
     );
   }
+
+  void _debounceOnChangedText(String text) {
+    _debounceOnChanged?.cancel();
+    _debounceOnChanged = Timer(const Duration(milliseconds: 300), () async {
+      if (mounted) {
+        _submitUpdateTaskDescription(text);
+      }
```

Comment:
Shouldn't need a mounted check, you're not doing a `setState`, if the user leaves the dialog within 300 ms of making a change, then the last change wouldn't be saved. And although it's highly unlikely, we can eliminate it by not having the mounted check. What do you think?

Reply:
Yeah that's reasonable. I added a commit to address this.